test/indexer/test_hca_indexer.py

```diff
@@ -1027,6 +1028,34 @@ def test_accessions_fields(self):
             self.assertEqual(['E-AAAA-00'], project['array_express_accessions'])
             self.assertEqual(['PRJNA000000'], project['insdc_study_accessions'])
 
+    def test_no_cell_count_contributions(self):
+        def assert_cell_suspension(expected_cell_count, expected_organ_parts, hits):
+            expected_cell_count_ = expected_cell_count
+            if expected_cell_count is None:
+                expected_cell_count = null_int.null_int
+            for hit in hits:
+                entity_type, aggregate = self._parse_index_name(hit)
+                if entity_type == 'projects' and aggregate:
+                    self.assertEqual(expected_cell_count,
```

Comment:
test would pass if there were no aggregate project hits

and I don't understand why you incrementally indexing the bundles without clearing the index. Each time this functions is called, the previous bundles' hits are asserted, too. How can this even work?

Reply:
consider a different approach, one that fingerprints the hits and asserts the fingerprint. I can explain in parking lot.